src/System.Runtime.Loader/tests/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <BuildConfigurations>
       netcoreapp;
+      uap;
```

Comment:
@tarekgh isn't uap running both? my builds did complain when one of the refs had only netcoreapp but adding uap has fixed the error and build was not complaining